project/src/main/java/com/google/sps/utilities/EventRanker.java

```diff
@@ -34,13 +41,19 @@
     return rankedEvents;
   }
 
-  private static Integer getEventScore(User user, Event event) {
-    Set<String> interestsAndSkills = new HashSet<>(user.getInterests());
-    interestsAndSkills.addAll(user.getSkills());
-    Set<String> labelsAndSkills = new HashSet<>(event.getLabels());
-    for (VolunteeringOpportunity opportunity : event.getOpportunities()) {
-      labelsAndSkills.addAll(opportunity.getRequiredSkills());
+  private static Map<Event, Double> getEventScoreMap(User user, Set<Event> events) {
+    List<Set<Pair<Event, Integer>>> allEvents =
```

Comment:
If the list always has 2 members, use Pair.

Reply:
Done

Reply:
Done